### PR TITLE
Convert to use transport directly

### PR DESCRIPTION
**Pull request by johnsimons**

### CODE REVIEW

File: src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+using Microsoft.Extensions.Logging;
+using NServiceBus.Transport;
+using ServiceControl.Transports;
+using Shared;
+
+internal class MonitoringThroughputHostedService(ITransportCustomization transportCustomization, TransportSettings transportSettings, ILogger<MonitoringThroughputHostedService> logger, MonitoringService monitoringService) : IHostedService
+{
+    private TransportInfrastructure? transportInfrastructure;
+
+    private async Task Handle(MessageContext message, CancellationToken cancellationToken)
```
Comment (jpalac) [inline src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs:13]: It doesn't work - this never gets hit - messages are not being processed from the queue.

File: src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
+namespace Particular.LicensingComponent.MonitoringThroughput;
+
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using NServiceBus.Transport;
+using ServiceControl.Transports;
+using Shared;
+
+internal class MonitoringThroughputHostedService(ITransportCustomization transportCustomization, TransportSettings transportSettings, ILogger<MonitoringThroughputHostedService> logger, MonitoringService monitoringService) : IHostedService
```
Comment (jpalac) [inline src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs:9]: We use BackgroundService for all the other ones - should we keep it in sync?

**Review by jpalac — CHANGES_REQUESTED** [review]
Doesn't seem to work - the messages sent from monitoring are not being received.

File: src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs (hunk 1 above)
Comment (johnsimons) [reply src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs:13]: I fixed that, we need to explicitly start the receiver

File: src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs (hunk 2 above)
Comment (johnsimons) [reply src/Particular.LicensingComponent/MonitoringThroughput/MonitoringThroughputHostedService.cs:9]: I looked at it, but unfortunately the signature methods of our transport don't really line up well with a single `run` method, hence the reason to do it that way.
This is the way the other are also done.

**Review by jpalac — APPROVED** [review]

**Pull request merged**